Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,106 @@
+
+On boot, heads will ask for the disk unlock passphrase that will release the Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the Key was created.  
+
+This option adds an extra layer of security.  The TPM will only release the Disk Unlock key if the measurements match and generally an attacker would not have the Disk Unlock passphrase.   If an attacker clones the drive and compromises the disk unlock passphrase he will still not have access to the decrypted data.  This is due to this passphrase being valid only on this laptop with this firmware and these measured files.  
+
+To mitigate recovery shell attacks, entering the recovery shell will modify the PCR and measurements. Also changes to kernel or initrd files on /boot will modify the measurements.  This will be detected as tampering.
+
+In this model the TPM enforces rate limits to prevent brute force attacks.
+
+#### Disable TPM Disk Unlock passphrase/key
+
+This feature may be turned off during configuration of heads.  The following will instruct heads not to ask for a disk unlock passphrase and to not store a disk unlock key in the TPM.  
+
+  CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
+
+
+### Using only a LUKS passphrase
+
+In this model heads is not responsible for any of the encryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
+
+This model is vulnerable to brute force attacks but does not lock the data to the hardware.
+
+### Security Dongle (aka smartcard)
+
+You may use a security dongle to manage the passphrase for the root LUKS partition.  This is also out of scope for heads even if using the same security dongle for verification such as in the [Pureboot](https://docs.puri.sm/PureBoot.html) model.  
+
+The first advantage to this approach is that the user only needs to type a PIN on boot rather than a passphrase.  Unlike the [heads TPM](#using-tpm-for-decryption-of-drives) model this model does not lock the data on disk to the hardware.  The hard drive may be moved to a different system and used with the security dongle and/or LUKS passphrase if it is still on the partition.  Also, the security dongle enforces limits which prevent brute force attacks.
+
+In this model a large amount of text or binary data is used as the passphrase for decrypting the LUKS container.  This complex passphrase is encyrpted using the private key associated with the security dongle and placed in a 'public' location such as /boot.  After heads boots the Operating System the kernel and initrd use the smartcard to decrypt the passphrase which is used to decrypt the LUKS root partition.  
+
```

Comment:
?

Reply:
describing one model of booting--which is the one I use.  I also have changed 'smartcard' to 'security dongle' in the latest docs. which might clear this up.  push in a few.

Reply:
removed duplicate sections and reworded to be more generic.